test/RelayServer.test.js

```diff
@@ -313,21 +312,15 @@ contract('RelayServer', function (accounts) {
 
       const expectedGasPrice = (await _web3.eth.getGasPrice()) * defunctRelayServer.gasPriceFactor
       const expectedBalance = await _web3.eth.getBalance(defunctRelayServer.address)
-      const chainId = await _web3.eth.getChainId()
-      const networkId = await _web3.eth.net.getId()
       assert.notEqual(defunctRelayServer.gasPrice, expectedGasPrice)
       assert.notEqual(defunctRelayServer.balance, expectedBalance)
-      assert.notEqual(defunctRelayServer.chainId, chainId)
```

Comment:
removed, since it implicitly assumes that `refreshStake()` can run without fully initializing the RelayServer .